### PR TITLE
Prop fill env

### LINKED ISSUE #524: Integrate prop filling code into build system

### PR DESCRIPTION
**Pull request by madeleinemc**
# Prop filling environment 

Fixes #524 

### Summary of changes
- Created the prop_fill environment in PlatformIO, which runs the prop filling code with gomspace pings (in src/prep).

After this gets approved, I'll add it to the Pre-Integration Procedures documentation.

### Testing
Builds successfully (pio run -e prop_fill)

### CODE REVIEW

File: src/prep/FillingSequence.ino [hunk 1]
```diff
@@ -0,0 +1,269 @@
+
```
Comment (tanishqaggarwal) [inline src/prep/FillingSequence.ino:1]: I would just get rid of this file to reduce confusion

File: configs/prop_fill_env.ini [hunk 2]
```diff
@@ -0,0 +1,8 @@
+; prop_fill_env
+;
+; Environment for prop filling/topoff
+[env:prop_fill]
+extends = teensy36
```
Comment (tanishqaggarwal) [inline configs/prop_fill_env.ini:5]: I would actually not extend teensy bc I don't know how all the additional flags in that environment affect this file. Could you use the base teensy framework that's specified in the platformIO documentation instead?

**Review by tanishqaggarwal — CHANGES_REQUESTED** [review]
The file compiles for me, I just want to be careful with what options we use for compilation.

**Comment by madeleinemc** [comment]
note that the most recent commit has only been tested on a teensy35 (on modified code) and not a teensy36 (but the code is for a teensy36)

**Review by tanishqaggarwal — APPROVED** [review]
🇺🇸 🚀 🇺🇸💥 🇺🇸

**Pull request merged**